Exercise08answers.R

```diff
@@ -0,0 +1,39 @@
+#Exercise 08 Answers
+
+#Question 1
+#Creating a function
+top<-function(r,line)
+  + {subf<-r[1:line,]
+  + return(subf)}

```

Comment:
The question does not ask you to write a function. Also there should not be "+" in here

Reply:
-0.25

Reply:
I have a further discussion with Dr Jones and he said this will be Ok
+0.25